scripts/imu_spawners/spawn_estimated_imu.py

```diff
@@ -192,7 +192,7 @@ def load_estimated_poses(filename):
 if __name__ == '__main__':
     # rospy.init_node("set_estimated_imu_positions")
     n_joint = 7
-    n_imu = 7
+    n_imu = 6
```

Comment:
Are we able to modify these two lines to read from the save data file?

Reply:
yes, I will do that.